ui/appui-react/src/appui-react/widget-panels/DockedBar.scss

```diff
@@ -2,10 +2,20 @@
  * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
  * See LICENSE.md in the project root for license terms and full copyright notice.
  *--------------------------------------------------------------------------------------------*/
-@import "~@itwin/core-react/lib/cjs/core-react/style/themecolors";
 
-.uifw-widgetPanels-statusBar {
-  > div {
-    background-color: $buic-background-2;
+.uifw-dockedBar {
+  background-color: var(--iui-color-background);
+  height: var(--iui-component-height);
+
+  &[data-placement="top"] {
+    border-block-end: 1px solid var(--iui-color-border);
+  }
+
+  &[data-placement="bottom"] {
+    border-block-start: 1px solid var(--iui-color-border);
+  }
+
+  label {
```

Comment:
@GerardasB  what do you think about having styling for all labels here. On one hand I am not a big fan of putting styling on all elements of label type, however, it is the easiest way to ensure that all labels within `StatusBar` and `ToolSettings` look the same. Besides, labels should only be used to describe input elements and nothing else so maybe it is fine to have this styling. What do you think?

Reply:
Let's avoid that since it's generally a bad idea to target an element you are not in control.

Reply:
Alright, I moved the margin to `uifw-default-label` class so at least all labels in tool settings will be the same.